examples/bri-3/src/bri/decorators/public-key.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  createParamDecorator,
+  ExecutionContext,
+  NotFoundException,
+} from '@nestjs/common';
+import { decodeJWT } from 'did-jwt';
+import { TOKEN_NOT_FOUND } from '../shared/constants';
+
+export const PublicKey = createParamDecorator(
```

Comment:
See my comments on accessing the context is a separate service exposing a simple method to fetch the logged in user information.

Reply:
TODO added